tracer/test/Datadog.Trace.Security.Unit.Tests/FuzzEncoder.cs

```diff
@@ -0,0 +1,193 @@
+// <copyright file="FuzzEncoder.cs" company="Datadog">
+// Unless explicitly stated otherwise all files in this repository are licensed under the Apache 2 License.
+// This product includes software developed at Datadog (https://www.datadoghq.com/). Copyright 2017 Datadog, Inc.
+// </copyright>
+
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Text;
+using Bogus;
+using Datadog.Trace.AppSec.Waf;
+using Datadog.Trace.AppSec.Waf.Initialization;
+using Datadog.Trace.AppSec.Waf.NativeBindings;
+using Datadog.Trace.Vendors.Newtonsoft.Json;
+using Datadog.Trace.Vendors.Newtonsoft.Json.Linq;
+using Xunit;
+
+namespace Datadog.Trace.Security.Unit.Tests
+{
+    public class FuzzEncoder
+    {
+        private static readonly JTokenType[] OpeningTypes = new[] { JTokenType.Array, JTokenType.Object };
+        private static readonly JTokenType[] TerminatingTypes = new[] { JTokenType.Integer, JTokenType.Float, JTokenType.String, JTokenType.Boolean, JTokenType.Null, JTokenType.Date, JTokenType.Guid, JTokenType.Uri, JTokenType.TimeSpan };
+        private readonly Random _rnd = new();
+        private readonly Faker faker = new("en");
+
+        [Fact]
+        [Trait("Category", "ArmUnsupported")]
```

Comment:
What's unsupported? I thought the WAF ran on ARM now?

Reply:
Not yet, there's a version WAF for ARM available, but we need to do the work to integrate the new binaries. I've logged this as jira ticket: https://datadoghq.atlassian.net/browse/APPSEC-2637